### PR TITLE
Remove GITHUB_USERNAME from the readme

### PR DESCRIPTION
**Pull request by jyn514**
It's no longer used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,11 +188,11 @@ cargo run -- build crate --local /path/to/source
 cargo run -- database add-directory <DIRECTORY> [PREFIX]
 
 # Updates github stats for crates.
```
Comment (syphar) [inline README.md:190]:
```suggestion
# Updates repository stats for crates.
```

File: README.md [hunk 2]
```diff
@@ -188,11 +188,11 @@ cargo run -- build crate --local /path/to/source
...
-# You need to set CRATESFYI_GITHUB_USERNAME, CRATESFYI_GITHUB_ACCESSTOKEN
+# You need to set CRATESFYI_GITHUB_ACCESSTOKEN
 # environment variables in order to run this command.
 # Set DOCSRS_GITLAB_ACCESSTOKEN to raise the rate limit,
```
Comment (syphar) [inline README.md:193]:
```suggestion
# Set DOCSRS_GITLAB_ACCESSTOKEN to raise the rate limit for GitLab repositories,
```

File: README.md [hunk 3]
```diff
@@ -188,11 +188,11 @@ cargo run -- build crate --local /path/to/source
...
 # or leave it blank to fetch repositories at a slower rate.
```
Comment (syphar) [inline README.md:194]:
```suggestion
# or leave it blank to fetch these repositories at a slower rate.
```

**Review by syphar — COMMENTED** [review]
That changed paragraph didn't fully make sense to me, I might be wrong :) 

And while we are touching it anyway?

File: README.md (hunk 3 above)
Comment (jyn514) [reply README.md:194]: Hmm, I don't think this makes things more clear. I would probably just say
```suggestion
# or leave it blank to fetch them at a slower rate.
```
now that we've specified "GitLab" on the line above.

**Review by jyn514 — COMMENTED** [review]
> That changed paragraph didn't fully make sense to me, I might be wrong :)

Hmm, can you expand on this? What was confusing?

**Review by syphar — APPROVED** [review]
now it's better :)

**Pull request merged**